src/test/groovy/com/twcable/grabbit/jcr/JCRNodeDecoratorSpec.groovy

```diff
@@ -27,11 +28,22 @@ import javax.jcr.nodetype.ItemDefinition
 import javax.jcr.nodetype.NodeDefinition
 import javax.jcr.nodetype.NodeType
 
+import static org.apache.jackrabbit.JcrConstants.JCR_CREATED
 import static org.apache.jackrabbit.JcrConstants.JCR_LASTMODIFIED
 import static org.apache.jackrabbit.JcrConstants.JCR_PRIMARYTYPE
 
 @SuppressWarnings("GroovyAssignabilityCheck")
 class JCRNodeDecoratorSpec extends Specification {
+    @Shared
```

Comment:
I had to check date returned from the method.. and it does calendar.getTime() which is a final method. So could not mock that. Moreover I had to use same instance in where block also, so I used @Shared here..